lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -517,14 +525,62 @@ MITIGATION_01_CRI_DCMAW:
       targetState: POST_DCMAW_SUCCESS_PAGE
     not-found:
       targetState: PYI_NO_MATCH
+      checkFeatureFlag:
+        f2fCimit:
+          targetState: MITIGATION_01_SUGGEST_F2F
+          checkIfDisabled:
+            f2f:
+              targetState: PYI_NO_MATCH
     access-denied:
       targetState: PYI_ANOTHER_WAY
+      checkFeatureFlag:
+        f2fCimit:
+          targetState: MITIGATION_01_SUGGEST_F2F
+          checkIfDisabled:
+            f2f:
+              targetState: PYI_ANOTHER_WAY
     temporarily-unavailable:
       targetState: PYI_ANOTHER_WAY
+      checkFeatureFlag:
+        f2fCimit:
+          targetState: MITIGATION_01_SUGGEST_F2F
+          checkIfDisabled:
+            f2f:
+              targetState: PYI_ANOTHER_WAY
     fail-with-no-ci:
       targetState: PYI_ANOTHER_WAY
+      checkFeatureFlag:
+        f2fCimit:
+          targetState: MITIGATION_01_SUGGEST_F2F
+          checkIfDisabled:
+            f2f:
+              targetState: PYI_ANOTHER_WAY
     enhanced-verification:
       targetState: PYI_ANOTHER_WAY
+      checkFeatureFlag:
+        f2fCimit:
+          targetState: MITIGATION_01_SUGGEST_F2F
+          checkIfDisabled:
+            f2f:
+              targetState: PYI_ANOTHER_WAY
```

Comment:
We should make sure that these match the routing from the same-session mitigation journey, and that the right ones get routed to F2F - see https://github.com/govuk-one-login/ipv-core-back/pull/1262/files#r1363817667

cc @lwebb-iig

Reply:
So here in case of event /enhanced-verification it get routed to F2F (MITIGATION_01_SUGGEST_F2F) but if F2F is disabled then will route to PYI_ANOTHER_WAY.

Reply:
I think we've agreed the way forward after this discussion this morning :)